application/app/Http/Requests/ActivateInstitutionUserRequest.php

```diff
@@ -4,12 +4,36 @@
 
 use App\Enums\InstitutionUserStatus;
 use App\Http\Requests\Traits\FindsInstitutionUsersWithAnyStatus;
+use App\Http\Resources\API\RoleResource;
 use App\Models\InstitutionUser;
 use App\Models\Role;
 use App\Rules\ModelBelongsToInstitutionRule;
 use Closure;
 use Illuminate\Foundation\Http\FormRequest;
+use OpenApi\Attributes as OA;
 
+#[OA\RequestBody(
+    request: self::class,
+    required: true,
+    content: new OA\JsonContent(
+        required: ['institution_user_id', 'notify_user', 'roles'],
+        properties: [
+            new OA\Property(
+                property: 'institution_user_id',
+                description: 'UUID of institution user to reactivate',
+                type: 'string',
+                format: 'uuid'
+            ),
+            new OA\Property(property: 'notify_user', type: 'boolean'),
+            new OA\Property(
+                property: 'roles',
+                type: 'array',
+                items: new OA\Items(ref: RoleResource::class),
```

Comment:
@DanBondarenko why the RoleResource is referenced under Request class documentation?

Reply:
@thenouan  Well spotted! That was a typo. Fixed with the latest commit.